src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -2244,22 +2225,11 @@ void testNumerics(SQLServerStatement stmt, String cekName, String[][] table, Str
         }
     }
 
-    SQLServerKeyVaultAuthenticationCallback authenticationCallback = new SQLServerKeyVaultAuthenticationCallback() {
-        // @Override
-        ExecutorService service = Executors.newFixedThreadPool(2);
-
-        public String getAccessToken(String authority, String resource, String scope) {
-
-            AuthenticationResult result = null;
-            try {
-                AuthenticationContext context = new AuthenticationContext(authority, false, service);
-                ClientCredential cred = new ClientCredential(applicationClientID, applicationKey);
-                Future<AuthenticationResult> future = context.acquireToken(resource, cred, null);
-                result = future.get();
-            } catch (Exception e) {
-                fail(TestResource.getResource("R_unexpectedException") + e.getMessage());
-            }
-            return result.getAccessToken();
-        }
-    };
+    private TokenCredential createTokenCredential() {
+        return new ClientSecretCredentialBuilder()
+                .clientId(applicationClientID)
+                .clientSecret(applicationKey)
+                .tenantId(tenantID)
```

Comment:
This tenantID will be null in CI mode.
Does the CI run live tests ? if so, we need to update yml config, to supply tenant ID as env var.